README.md

```diff
@@ -26,83 +25,84 @@ For more information check out [metabase.com](https://metabase.com/)
 
 ## Supported databases
 
-- Postgres
-- MySQL
+- Amazon Redshift
 - Druid
-- SQL Server
-- Redshift
-- MongoDB
+- Google Analytics
 - Google BigQuery
-- SQLite
 - H2
+- MongoDB
+- MySQL
 - Oracle
-- Vertica
+- Postgres
 - Presto
 - Snowflake
 - SparkSQL
+- SQL Server
+- SQLite
+- Vertica
 
-Don't see your favorite database? File an issue to let us know.
+Don't see your favorite database? Check the list of [community supported drivers](https://www.metabase.com/docs/latest/developers-guide-drivers.html#how-to-use-a-community-built-driver).
 
 ## Installation
 
-Metabase can be run just about anywhere so checkout our [Installation Guides](https://metabase.com/docs/latest/operations-guide/start.html#installing-and-running-metabase) for detailed instructions for various deployments. Here's the TLDR:
+Metabase can be run just about anywhere so checkout our [Installation Guides](https://www.metabase.com/docs/latest/operations-guide/installing-metabase.html) for detailed instructions for various deployments. Here's the TLDR:
 
 ### Docker
 
 To run Metabase via Docker, just type
 
-```sh
+```
 docker run -d -p 3000:3000 --name metabase metabase/metabase
 ```
 
-### JVM Jar
+### JAR-file
 
-To run the jar you will need to have a Java Runtime installed. As a quick check to see if you system already has one, try
+To run Metabase via JAR-file, you will need to have a [Java Runtime Environment](https://www.metabase.com/docs/latest/operations-guide/java-versions.html) installed on your system.
 
-```sh
+As a quick check to see if your system already has Java installed and it's version details, try running this command from a terminal:
```

Comment:
I wonder if we should include this section on java versions, as it's imprecise (what does "similar to this" mean), and unclear (" the Java release date is more than a few months old, then you need to install or update Java."). Maybe we should just recommend "AdoptOpenJDK+Hotspot", but clarify that Metabase will work with Java LTS version 8 or higher (per [cam's comments on 12433](https://github.com/metabase/metabase/issues/12433)).

Reply:
I'm all for removing that section from `README.md` and `operations-guide/running-the-metabase-jar-file.md` - just replaced the current ones with the new one in `operations-guide/java-versions.md`, so they were at least matching.
Was avoiding version numbers, so we don't have to update that.